src/spec/genre_spec.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../Classes/genre'
+
+describe Genre do
+  context 'Unit tests for Genre Class' do
+    it 'should create a new genre when the class is instantiated' do
```

Comment:
Please create separate 'it' tests for checking whether the genre object is instantiated properly, and another one for checking the items and so on. (you can checkout my tests)